module/Order/src/Order/Entity/Order.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Order\Entity;
+
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="orders")
+ */
+
+class Order {
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue
+     **/
+    protected $id;
+
+    /**
+     * @ORM\Column(type="integer")
+     **/
+    protected $order_no;
```

Comment:
we need to discuss about variable naming convention. Our database standard asks us to use snake_case and PHP standard suggests camelCase. Let's leave it as it is for now.